frontend/src/app/main/ui/workspace/colorpicker.cljs

```diff
@@ -102,6 +102,11 @@
         shared-libs (mf/deref refs/workspace-libraries)
         recent-colors (mf/deref refs/workspace-recent-colors)
 
+        picking-color? (mf/deref refs/picking-color?)
+        picked-color (mf/deref refs/picked-color)
+        picked-color-select (mf/deref refs/picked-color-select)
+        picked-shift? (mf/deref refs/picked-shift?)
+
```

Comment:
Seems like the picker refs are only used here, in this case i think they should be defined here in this namespace instead of on refs. Refs is for commonly used refs used in many sites (as a general rule). What do you think about this?

Reply:
Yes, I think it's better. I've changed it